2.6/Dockerfile

```diff
@@ -1,10 +1,19 @@
 FROM centos:centos7
-MAINTAINER SoftwareCollections.org <sclorg@redhat.com>
+LABEL MAINTAINER SoftwareCollections.org <sclorg@redhat.com>
```

Comment:
I don't think we have guidelines written anywhere, the practice is that we use it for Fedora and CentOS images (the mailing list), but not for RHEL images, but I don't see this written somewhere either. A quick look-up showed that lowercase `maintainer` seems preferred (https://github.com/projectatomic/ContainerApplicationGenericLabels/blob/master/vendor/redhat/labels.md and also mentioned once in the Fedora Guidelines)
and it would make sense to me to handle the `maintainer` label the same as other labels..